web_sheet_widescreen/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Web - Sheets for Widescreen",
+    "version": "10.0.1.0.1",
+    "author": "Mark Robinson",
```

Comment:
@markrobinson85 author like [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L12)

Add 'website' key like [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

Reply:
and OCA as author